Added CSS transforms to work on SVG

gregwhitworth opened the pull request:
This is a common request, I've had it twice this week on Twitter alone. Adding this to status for web devs to be able to keep track of its relative ranking.

gregwhitworth committed:
Added CSS transforms to work on SVG

molant commented:
@gregwhitworth @jacobrossi should we add the status for Firefox and Chrome if chromestatus doesn't have this information?

gregwhitworth committed:
Added other vendor support, uservoiceid, and changed transitions title to not include the word animations

gregwhitworth commented:
@molant Good catch, added that in. Also removed the "& animations" from the CSS transitions feature as people often assume "Animations" includes transforms (which it does btw) to avoid confusion.

jacobrossi commented:
LGTM
